webapi/src/main/java/org/sakaiproject/webapi/controllers/DashboardController.java

```diff
@@ -103,38 +103,44 @@ public class DashboardController extends AbstractSakaiApiController {
     @PostConstruct
     public void init() {
 
+        boolean tasksEnabled = serverConfigurationService.getBoolean("dashboard.tasks.enabled", false);
+
         // Load up all the available widgets, from properties
-        String[] courseWidgetsArray = serverConfigurationService.getStrings("dashboard.course.widgets");
-        if (courseWidgetsArray == null) {
-            courseWidgetsArray = new String[] {"tasks", "announcements", "calendar","forums", "grades" };
+        courseWidgets = serverConfigurationService.getStringList("dashboard.course.widgets", null);
+        if (courseWidgets.isEmpty()) {
```

Comment:
will this survive a null?

Reply:
Yep. getStringList returns an empty list if you supply a null default :)

Reply:
haha ok thanks

Reply:
I've updated the javadoc on ServerConfigurationService to explain that

Reply:
that was nice of you. cheers.